components/dialog/dialog-styles.js

```diff
@@ -119,19 +119,23 @@ export const dialogStyles = css`
 	}
 
 	@media (max-width: 615px) {
-
 		.d2l-dialog-header {
 			padding: 14px 20px 16px 20px;
 		}
-
 		.d2l-dialog-content {
 			padding: 0 20px;
 		}
-
 		.d2l-dialog-footer {
 			padding: 20px;
 		}
-
 	}
 
+	@media (prefers-reduced-motion: reduce) {
```

Comment:
Does it ever make sense to inverse this? i.e. only apply the transition in the first place if the value is `no-preference`. Or does that not work in browser who don't support this (IE11).

Reply:
Hmmm that's a good question.  Both old-Edge and IE11 both return `false` for these media queries, regardless of `reduce` or `no-preference`, so I think in order for animations to be the default, we need to write the queries to opt out by using `reduce`.  In my testing, the other browsers do translate `no-preference` -> `true` though.